com.unity.render-pipelines.high-definition/Editor/Lighting/LightUnit/LightUnitSlider.cs

```diff
@@ -0,0 +1,316 @@
+
+            iconRect = baseRect;
+            iconRect.x += sliderRect.width + SliderConfig.k_IconSeparator;
+            iconRect.width = EditorGUIUtility.singleLineHeight;
+        }
+
+        void DoSliderMarker(Rect rect, float position, float value, string tooltip)
+        {
+            const float width  = SliderConfig.k_MarkerWidth;
+            const float height = SliderConfig.k_MarkerHeight;
+
+            var markerRect = rect;
+            markerRect.width  = width;
+            markerRect.height = height;
+
+            // Vertically align with slider.
+            markerRect.y += (EditorGUIUtility.singleLineHeight / 2f) - 1;
+
+            // Horizontally place on slider.
+            const float halfWidth = width * 0.5f;
+            markerRect.x = rect.x + rect.width * position;
+
+            // Center the marker on value.
+            markerRect.x -= halfWidth;
+
+            // Clamp to the slider edges.
+            float min = rect.x;
+            float max = (rect.x + rect.width) - width;
+            markerRect.x = Mathf.Clamp(markerRect.x, min, max);
+
+            // Draw marker by manually drawing the rect, and an empty label with the tooltip.
+            EditorGUI.DrawRect(markerRect, Color.white);
+
+            // Scale the marker tooltip for easier discovery
+            const float markerTooltipRectScale = SliderConfig.k_MarkerTooltipScale;
+            var markerTooltipRect = markerRect;
+            markerTooltipRect.width  *= markerTooltipRectScale;
+            markerTooltipRect.height *= markerTooltipRectScale;
+            markerTooltipRect.x      -= (markerTooltipRect.width  * 0.5f) - 1;
+            markerTooltipRect.y      -= (markerTooltipRect.height * 0.5f) - 1;
+            EditorGUI.LabelField(markerTooltipRect, GetLightUnitTooltip(tooltip, value));
+        }
+
+        void DoIcon(Rect rect, GUIContent icon, float range)
+        {
+            var oldColor = GUI.color;
+            GUI.color = Color.clear;
+            EditorGUI.DrawTextureTransparent(rect, icon.image);
+            GUI.color = oldColor;
+
+            EditorGUI.LabelField(rect, GetLightUnitTooltip(icon.tooltip, range));
+        }
+
+        void DoThumbTooltip(Rect rect, float position, float value, string tooltip)
+        {
+            const float size = SliderConfig.k_ThumbTooltipSize;
+            const float halfSize = SliderConfig.k_ThumbTooltipSize * 0.5f;
+
+            var thumbMarkerRect = rect;
+            thumbMarkerRect.width  = size;
+            thumbMarkerRect.height = size;
+
+            // Vertically align with slider
+            thumbMarkerRect.y += halfSize - 1f;
+
+            // Horizontally place tooltip on the wheel,
+            thumbMarkerRect.x  = rect.x + (rect.width - size) * position;
+
+            EditorGUI.LabelField(thumbMarkerRect, GetLightUnitTooltip(tooltip, value));
+        }
+
+        GUIContent GetLightUnitTooltip(string baseTooltip, float value)
```

Comment:
This sounds like a useful utility here. Maybe you can extract a static function to convert a value to a readable human string (float value) -> string.
And then use it here.

Also, maybe we can use existing API like https://github.com/Humanizr/Humanizer#metric-numerals or similar?

Reply:
Hey @fredericv-unity3d, due to artist request we now just display the whole value here instead of abbreviating it into shorter terms. So I will mark resolved for now